### ISSUE TITLE
Consistency regarding format-message and t

### ISSUE THREAD

**Issue by vvohra** [ember-intl/ember-intl#480]
- [x] I am on the [latest](https://github.com/jasonmit/ember-intl/releases/latest) ember-intl version
- [x] I have searched the [issues](https://github.com/jasonmit/ember-intl/issues) of this repo and believe that this is not a duplicate

### Environment

- Ember Version: 2.8
- Ember CLI Version: 2.6.3
- Ember Intl Version: 2.24.1

### Question

So:
Service methods: `formatMessage` does NOT accept a translation key, but `t` does

Helper methods: both `formatMessage` and `t` accept translation keys.

Is that intentional? Should it be consistent?

**Comment by jasonmit** [comment]
Legacy logic that was moved forward while I waited to see how ember-i18ns API settled around formatting html-safe translations.

`format-message` helper in 3.0 will take a string literal, to match the service API.  I first need to deprecate the method in 2.x to encourage anyone using it to converting to the `t` helper.

The `t` helper will take an `htmlSafe` attribute to signify returning an html safe string.  The `t` method on the service will also respect this attribute.
